runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2718,22 +2756,24 @@ MM_CopyForwardScheme::getNextScanCache(MM_EnvironmentVLHGC *env, UDATA preferred
 		}
 
 		if (NULL == cache) {
-			omrthread_monitor_enter(_scanCacheMonitor);
-			_scanCacheWaitCount += 1;
+			omrthread_monitor_enter(*_scanCacheMonitorPtr);
```

Comment:
is getNextScanCache() still used? same for getNextScanCacheOnNode.

I hope to remove them, if  not now, than at least when we prove the hybrid scheme is stable.

Reply:
removed